### PR TITLE
CO-1627 add fin aid tracking requirements

### PR DESCRIPTION
**Pull request by tsoliangwu0130**

### CODE REVIEW

File: src/api/v1/serializers/onbase-serializer.js [hunk 1]
```diff
@@ -60,4 +65,43 @@ const serializeAdmission = (rawRows, osuId) => {
...
+ * A function to serialize raw financial aid data
+ *
+ * @param {object[]} rawRows Raw data rows from data source
+ * @param {string} osuId OSU ID
+ * @returns {object} Serialized resources data
+ */
+const serializeFinancialAid = (rawRows, osuId) => {
+  const rawFinancialAid = {
+    osuId,
+    type: 'financialAid',
```
Comment (Ruefa) [inline src/api/v1/serializers/onbase-serializer.js]: Is this string different from `financialAidResourceType`?

**Review by Ruefa — CHANGES_REQUESTED** [review]

File: src/api/v1/serializers/onbase-serializer.js (hunk 1 above)
Comment (tsoliangwu0130) [reply src/api/v1/serializers/onbase-serializer.js]: Good catch, fixed.

**Review by Ruefa — APPROVED** [review]

**Review by Ruefa — CHANGES_REQUESTED** [review]
I'm getting a 500 error when running `https://localhost:4000/api/v1/onbase/financial-aid/932776660?financialAidYear=2018`

**Review by Ruefa — APPROVED** [review]

File: openapi.yaml [hunk 2]
```diff
@@ -137,6 +205,42 @@ parameters:
       required:
         - data
       additionalProperties: false
+  financialAidPatchBody:
```
Comment (Ruefa) [inline openapi.yaml:210]: Shouldnt this be required? It looks like the admissionsPatchBody isnt required either.

File: openapi.yaml [hunk 3]
```diff
@@ -137,6 +205,42 @@ parameters:
...
+              $ref: '#/definitions/FinancialAidType'
+            attributes:
+              properties:
+                onBaseDocumentType:
+                  type: string
+                  description: OnBase document type
+                  example: FA Last Day Attended
+                financialAidYear:
+                  type: string
+                  description: Search terms by financial aid year
```
Comment (Ruefa) [inline openapi.yaml]: In the description we should point out the financialAidYear is actually two years. When I read the example `1819` I read that as the year 1819 and not 2018-2019

**Review by Ruefa — CHANGES_REQUESTED** [review]

File: openapi.yaml (hunk 2 above)
Comment (tsoliangwu0130) [reply openapi.yaml:210]: Good catch, fixed.

File: openapi.yaml (hunk 3 above)
Comment (tsoliangwu0130) [reply openapi.yaml]: Sounds good, fixed.

**Review by Ruefa — APPROVED** [review]

**Pull request merged**